README.md

```diff
@@ -218,11 +223,16 @@ type InputMetadata = {
   text: string;
 };
 
+type ImageMetadata = {
+  src: string;
```

Comment:
Thoughts on this one @spennyp ?
I am thinking to set this to `target` instead, so there is some balance with the new `link` and `mint` actions.

Or we think `src` is better for an image?